dev-tools/omdb/src/bin/omdb/db.rs

```diff
@@ -425,6 +442,92 @@ enum NetworkCommands {
     ListVnics,
 }
 
+#[derive(Debug, Args)]
+struct RegionArgs {
+    #[command(subcommand)]
+    command: RegionCommands,
+}
+
+#[derive(Debug, Subcommand)]
+enum RegionCommands {
+    /// List all regions
+    List(RegionListArgs),
+
+    /// Find what is using a region
+    UsedBy(RegionUsedByArgs),
+
+    /// Find deleted volume regions
+    FindDeletedVolumeRegions,
+}
+
+#[derive(Debug, Args)]
+struct RegionListArgs {
+    /// Print region IDs only
+    #[arg(short)]
+    id_only: bool,
+}
+
+#[derive(Debug, Args)]
+struct RegionUsedByArgs {
+    region_id: Vec<Uuid>,
+}
+
+#[derive(Debug, Args)]
+struct SagaArgs {
+    #[command(subcommand)]
+    command: SagaCommands,
+}
+
+#[derive(Debug, Subcommand)]
+enum SagaCommands {
+    /// List all sagas
+    List(SagaListArgs),
+
+    /// List sagas that failed to unwind
+    Stuck,
+
+    /// Show the execution of a saga
+    Show(SagaShowArgs),
+
+    /// List sagas that encountered an error and successfully unwound
+    Unwound,
+
+    /// Show any failing nodes
+    Failing,
+
+    /// List any sagas which overlap execution with this one
+    Overlapping(SagaOverlappingArgs),
+
+    /// Show multiple saga executions along a timeline
+    Interleave(SagaInterleaveArgs),
```

Comment:
These sound pretty neat!  Do you have example output?  (I guess it would be hard to write an automated test for this?)